app/client/components/Clipboard.js

```diff
@@ -143,24 +150,24 @@ Clipboard.prototype._doContextMenuCut = function() {
   this._copyToClipboard(pasteObj, 'cut');
 };
 
-Clipboard.prototype._setCBdata = function(pasteObj, clipboardData) {
+Clipboard.prototype._setCBdata = function(pasteObj, clipboardData, includeColHeaders) {
```

Comment:
Can you remove this parameter, I haven't found any code that is passing it as true

Reply:
Done.